WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -227,6 +230,7 @@ final class WordPressComRestAPIResponseSerializer: AFJSONResponseSerializer
         let extraStatusCodes = NSMutableIndexSet(indexSet: self.acceptableStatusCodes!)
         extraStatusCodes.addIndexesInRange(NSRange(400...500))
         self.acceptableStatusCodes = extraStatusCodes
+        self.readingOptions = [.AllowFragments]
```

Comment:
That assumption of being a dictionary is not true, we should always check the type, even without turning this on you could also have an array has top level object in JSON. I think this is a better solution because I don't think that the JSON spec mandates an Dictionary or Array as top level object. https://www.ietf.org/rfc/rfc4627.txt